BoxContentSDK/BoxContentSDK/Models/BOXRecentItem.m

```diff
@@ -0,0 +1,56 @@
+//
+//  BOXRecentItem.m
+//  Pods
+//
+//  Created by Andrew Dempsey on 12/19/16.
+//
+//
+
+#import "BOXRecentItem.h"
+
+@implementation BOXRecentItem
+
+- (instancetype)initWithJSON:(NSDictionary *)JSONData
+{
+    if (self = [super initWithJSON:JSONData]) {
+        NSDictionary *itemDictionary = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyItem
+                                                                      inDictionary:JSONData
+                                                                   hasExpectedType:[NSDictionary class]
+                                                                       nullAllowed:NO];
+
+
+        if ([itemDictionary[BOXAPIObjectKeyType] isEqualToString:BOXAPIItemTypeFile]) {
```

Comment:
Do you think handling bookmarks would be misleading? It should be safe to do and protect us for when they'll be supported... Not handling folders makes sense since that is a completely different category of thing